dlp/inspect_content.py

```diff
@@ -648,6 +769,16 @@ def callback(message):
              'include "FIRST_NAME", "LAST_NAME", "EMAIL_ADDRESS". '
              'If unspecified, the three above examples will be used.',
         default=['FIRST_NAME', 'LAST_NAME', 'EMAIL_ADDRESS'])
+    parser_gcs.add_argument(
+        '--custom_dictionaries', action='append',
+        help='Strings representing comma-delimited lists of dictionary words'
```

Comment:
Is this intended to be a single string of comma-delimited words, or multiple strings of one word each, or multiple comma-delimited strings?

Reply:
It is intended to be multiple strings, each of which is a string of comma-delimited words; I have updated the help message to make this more clear. This is how the other code samples are written.